documentation/specs/Card.md

```diff
@@ -0,0 +1,194 @@
+# `Card` Component Specification
```

Comment:
**question**: does the checked card variation thats shown in figma have any significance to the content that the card renders or is it just a use case example and not really a stateful variation of some sorts?